org.opent2t.sample.hub.superpopular/com.hue.hub/js/manifest.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest>
```

Comment:
Looks like this should be the same as nest not wink.  based on the hue onboarding PR

---
In reply to: [80634569](https://github.com/openT2T/translators/pull/96#discussion_r80634569) [](ancestors = 80634569)